packages/apps-config/src/api/spec/index.ts

```diff
@@ -120,7 +120,6 @@ const spec: Record<string, OverrideBundleDefinition> = {
   'bifrost-parachain': bifrostParachain,
   'bitcountry-node': bitcountry,
   'bitcountry-parachain': bitcountryParachain,
-  'btc-parachain': polkabtc,
```

Comment:
Possibly this - on non-upgraded chains (i.e. where the spec_name has not been changed), the types won't be injected at all. It will only match the new name below.

Would suggest leaving this line in. (just `: interbtc`

Reply:
Thank you! We're soon to re-genesis and are in this in-between phase with the chainspec names